src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery firstBall = new Lottery();
+        Lottery secondBall = new Lottery();
+        Lottery thirdBall = new Lottery();
+        Lottery[] array = new Lottery[]{firstBall, secondBall, thirdBall};
+        for (Lottery i: array) {
+            System.out.println(i.getRandomBall().toString());
+        }
```

Comment:
You should create just 1 instance of Lottery and invoke `getRandomBall()` method 3 times
also when passing a value to `System.out.println()` you dont need to explicitly call `toString()` it will be called automatically